src/coordinator/services/m3coordinator/main/main.go

```diff
@@ -142,6 +142,9 @@ func parseFlags(logger *zap.Logger) *m3config {
 	a.Flag("query.port", "Address to listen on.").
 		Default("0.0.0.0:7201").StringVar(&cfg.listenAddress)
 
+	a.Flag("query.resolution", "Resolution for queries.").
+		Default("15s").DurationVar(&cfg.resolution)
```

Comment:
Do you always need resolution now that you can return unconsolidated datapoints?  If so could we default this to 0 perhaps so users don't need to configure this?

Reply:
we don't need resolution anymore at the storage layer, that was an old artifact. I'll remove it